amida-tech/orange-api
# Hotfix: Enable Plus Sign in Email

## hotfix/enable-plus-sign-in-email
The "+" sign is now a valid character in email addresses.

## Branch Setup
None.

Note (just for clarity): No changes to the amida-auth-microservice required.

## Related Tickets
* [ORANGE-477](https://jira.amida-tech.com/browse/ORANGE-477)

## Screenshots
N/A

## Test Cases
1. Create a user that has a + sign in the email address. It should work, and, as is the norm for that process, it should also automatically log you in.
2. Login as a user who has a + sign in the email address. It should work.

## Readiness
1. [ N/A ] This PR has full test coverage
2. [ N/A ] This PR passess all tests
3. [ Yes ] This PR has no linting errors
4. [ Yes ] This PR has been updated to include all changes from develop
5. [ Yes ] This PR has been tested in iOS *and* Android
6. [ N/A ] This PR's changes to configration files for an instance's runtime have been documented
7. [ N/A ] This PR's required changes outside of the scope of this repository have been documented in this pull request
